activiti-cloud-services-org/activiti-cloud-services-org-service/src/main/java/org/activiti/cloud/services/organization/validation/extensions/ProcessExtensionsTaskMappingsValidator.java

```diff
@@ -50,16 +50,26 @@ public ProcessExtensionsTaskMappingsValidator(Set<TaskMappingsValidator> taskMap
 
     @Override
     public Stream<ModelValidationError> validateExtensions(Extensions extensions,
-                                                 BpmnProcessModelContent bpmnModel,
-                                                 ValidationContext validationContext) {
-        Set<FlowNode> availableTasks = bpmnModel.findAllNodes();
+                                                           BpmnProcessModelContent bpmnModel,
+                                                           ValidationContext validationContext) {
+
+        Set<FlowNode> availableTasks = bpmnModel.findAllNodes(
+                Task.class,
+                CallActivity.class,
+                StartEvent.class,
+                IntermediateCatchEvent.class,
+                EndEvent.class,
+                BoundaryEvent.class,
+                ThrowEvent.class);
+
+
         return extensions.getVariablesMappings().entrySet()
                 .stream()
                 .flatMap(taskMapping -> validateTaskMapping(bpmnModel.getId(),
-                                                            taskMapping.getKey(),
```

Comment:
It looks like we need to have the same rules about formatting, otherwise, we will keep changing this code

Reply:
I totally agree, the whole repo is a bit of a mess when it comes to code style

Reply:
Which is the code style/formatting rule that you refer here?

Reply:
@davidcanonieto the whole repo respects https://github.com/Activiti/activiti-build/tree/develop/ide-configuration/intellij-configuration/code-style, which was the required code style from the beginning. At least, I did respected it :)

Reply:
Apply that code-style and it should be good to merge.

Reply:
It should be good now, I don't know if Intellij needs a special configuration in order to pick up the code-style. Does anybody know?

Reply:
Here, under Recommended: import our code style point: https://github.com/Activiti/Activiti#configuring-intellij

Reply:
Also, please apply it to `ModelValidationControllerIT` class too